packages/replayio/src/commands/info.ts

```diff
@@ -0,0 +1,29 @@
+import { name as packageName, version as packageVersion } from "../../package.json";
+import { registerCommand } from "../utils/commander/registerCommand";
+import { exitProcess } from "../utils/exitProcess";
+import { getCurrentRuntimeMetadata } from "../utils/initialization/getCurrentRuntimeMetadata";
+import { parseBuildId } from "../utils/installation/parseBuildId";
+import { dim, highlight } from "../utils/theme";
+
+registerCommand("info", { requireAuthentication: false })
+  .description("Display info for installed Replay dependencies")
+  .action(info);
+
+async function info() {
+  console.log(`Currently using ${highlight(`${packageName}@${packageVersion}`)}`);
+
+  const metadata = getCurrentRuntimeMetadata("chromium");
+  if (metadata) {
+    const { buildId, nativeVersion } = metadata;
+
+    const { releaseDate } = parseBuildId(buildId);
+
+    console.log("\nReplay Chromium");
+    console.log(`• Release date: ${highlight(releaseDate.toLocaleDateString())}`);
+    if (nativeVersion) {
+      console.log(`• Native version: ${highlight(nativeVersion)}`);
```

Comment:
q: why do we call this a "native" version?

Reply:
I couldn't think of a better name for it. Original version? Base version? Forked version?

Reply:
A fork version could work - it makes sense and its readable (I assume that this is like a version from which this particular chromium fork is based on). A base version is OK too. I guess this is mostly useful for us anyway so it doesn't matter all that much.